beniget/beniget.py

```diff
@@ -193,9 +193,13 @@ class DefUseChains(ast.NodeVisitor):
     c -> (c -> (Call -> ()))
     """
 
-    def __init__(self):
+    def __init__(self, filename=None):
+        """
+            - filename: str, included in error messages if specified
+        """
         self.chains = {}
         self.locals = defaultdict(list)
+        self.filename = "{}:".format(filename) if filename else ""
```

Comment:
Sorry for being picky, but I'd rather have the formatting choice (adding `:`) be done in `unbound_identifier`. Likewise, keep `filename` to `None` and leave all the decision to  `unbound_identifier.

Thanks!

Reply:
OOpps commenting an obsolete change, sorry.